caffe2/python/tutorials/py_gen/MNIST.py

```diff
@@ -94,12 +108,18 @@ def DownloadResource(url, path):
 
 # > If the database wasn't found in the last step, [download the MNIST lmdb database](https://download.caffe2.ai/databases/mnist-lmdb.zip) or review the [datasets and databases notebook](https://github.com/caffe2/caffe2/blob/master/caffe2/python/tutorials/MNIST_Dataset_and_Databases.ipynb) on how to create the database from the MNIST dataset.
 
-# We will be using the `ModelHelper` class to represent our main model and using `brew` module and `Operators` to build our model. `brew` module has a set of wrapper functions that automatically separates the parameter intialization and the actual computation into two networks. Under the hood, a `ModelHelper` object has two underlying nets, `param_init_net` and `net`, that keeps record of the initialization network and the main network respectively.
+# We will be using the `ModelHelper` class to represent our main model and using `brew` module and normal Caffe2 operators to build our model. 
+# 
+# When you do `model.MyOperator` or `model.net.MyOperator` its going to do the same thing - add corresponding operator to model.net. 
```

Comment:
`model.MyOperator` is syntactic sugar for `model.net.MyOperator`, which adds the corresponding `MyOperator` operator to `model.net`.